tests/improver-spot-extract/02-nearest.bats

```diff
@@ -39,24 +39,21 @@
   # for extracting data.
   run improver spot-extract \
       "$IMPROVER_ACC_TEST_DIR/spot-extract/basic/nearest_diagnostics.json" \
+      "$IMPROVER_ACC_TEST_DIR/spot-extract/basic/temperature_at_screen_level.nc" \
       "$IMPROVER_ACC_TEST_DIR/spot-extract/basic" \
-      "$IMPROVER_ACC_TEST_DIR/spot-extract/basic" \
+      $TEST_DIR \
       --diagnostics temperature \
       --latitudes 10 20 30 40 50 60 \
       --longitudes 0 0 0 0 0 0 \
       --altitudes 0 1 2 3 4 5 \
```

Comment:
I think you need to remove the \ to get this to work

Reply:
Removed.